src/python/Python-2.7.10/setup.py

```diff
@@ -807,7 +807,8 @@ def detect_modules(self):
         search_for_ssl_incs_in = [
                               '/usr/local/ssl/include',
                               '/usr/sfw/include/',
-                              '/usr/contrib/ssl/include/'
+                              '/usr/contrib/ssl/include/',
+                              '/usr/local/opt/openssl/include'
```

Comment:
I know that this is how the current code is, but I think that for new changes we need a comment to describe why we need each separate path... so if later we refactor something or we remove a supported platform we will know that is safe to remove a path